README.md

```diff
@@ -1,4 +1,31 @@
 # PlaceRecognitionDB
 PlaceRecognitionDB is a tool for creating optimally sized databases (containing the minimum number of frames covering the scene) for place recognition task from RGBD and LiDAR data.
 
-The goal of the project is to create a tool with an optimal compression strategy for any RGBD and LiDAR data.
+The tool supports several basic methods for creating databases, as well as the DominatingSet method for creating optimal databases.
+
+## Datasets format
+To use the tool, your data must be in a specific format.
+* Color images in any format.
+* Depth images corresponding to color images in 16-bit grayscale format or point clouds in `.pcd` format.
```

Comment:
Should the corresponding color and depth images have the same name or is the correspondence based on some other attribute?